### PR TITLE
Fix/review exempt list

### LINKED ISSUE #1921: Review exempt transactions
add mint transaction to be exempt
remove healthcheck transactions from the list
remove add_miner  add_sharder and sharder_keep  from the list

### PR DESCRIPTION
**Pull request by devyetii**
## Fixes
#1921 
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/smartcontract/storagesc/allocation.go [hunk 1]
```diff
@@ -323,6 +323,8 @@ func (sc *StorageSmartContract) newAllocationRequestInternal(
 			AllocationId: sa.ID,
 			Delta:        bSize(request.Size, request.DataShards),
 		})
+
+		emitUpdateBlobber(b, balances)
```
Comment (dabasov) [inline code/go/0chain.net/smartcontract/storagesc/allocation.go]: why do we need to this call?

**Review by dabasov — DISMISSED** [review]

**Comment by peterlimg** [comment]
@dabasov I'm not sure if we should remove health check and `add_miner`, `add_sharder`, etc from the list as when I do txn fee PRs tests, those txns will fail to work due to the fee requirement.

**Comment by dabasov** [comment]
> @dabasov I'm not sure if we should remove health check and `add_miner`, `add_sharder`, etc from the list as when I do txn fee PRs tests, those txns will fail to work due to the fee requirement.

@peterlimg we discussed it with @guruhubb. all providers will have balance and will be refreshed periodically. the problem with free health checks is that network can be easily DDOSed, we should not have any exempt transactions at all, after the mainnet we will make mint fees to be payed from the minted amount

**Review by peterlimg — APPROVED** [review]
LGTM

**Pull request merged**